### PR TITLE
Fix for #2691

### PR DESCRIPTION
**Pull request by SvePu**
This could be a fix for issue #2691 - please review

### CODE REVIEW

File: admin/modules/user/groups.php [hunk 1]
```diff
@@ -673,6 +673,23 @@
 		{
 			$errors[] = $lang->error_missing_title;
 		}
+
```
Comment (Shade-) [inline admin/modules/user/groups.php]: Please remove the hidden tab chars.

File: admin/modules/user/groups.php [hunk 2]
```diff
@@ -815,6 +832,23 @@
...
```
Comment (Shade-) [inline admin/modules/user/groups.php]: Also here.

File: admin/modules/user/groups.php [hunk 3]
```diff
@@ -673,6 +673,23 @@
 		{
 			$errors[] = $lang->error_missing_title;
 		}
+		
+		if ($mybb->input['title'])
```
Comment (Shade-) [inline admin/modules/user/groups.php]: Conditionals and round brackets should not be separated with a space. Applies to the whole PR.

**Comment by SvePu** [comment]
@Shade- Thx for reviewing .... I removed the hidden tab chars & extra space

**Comment by Shade-** [comment]
As discussed within #2691, this PR is invalid.

**Pull request closed without merging**